engine/src/object.h

```diff
@@ -896,7 +896,10 @@ class MCObject :
 
 	MCObject *getparent() const
 	{
-		return parent;
+        if (parent)
+            return parent;
+        else
+            return NULL;
 	}
```

Comment:
Yeah, I was meaning to ask how best to handle this.

The parent is an object handle, which will trigger and assertion failure when trying to fetch a pointer to the original object if the original object is not valid.

In the case of setting the home stack during startup (MCDispatch::changehome), the parent of the new home stack is not set, which triggers the assertion failure.